m4a_hq_mixer.s

```diff
@@ -705,7 +699,7 @@ C_select_highspeed_codepath:
     /* This loads the needed code to RAM */
     STR     R0, previous_fast_code
     LDMIA   R0, {R0-R2, R8-R10}             @ load 6 opcodes
-    ADR     LR, fast_mixing_instructions
+    ADR     LR, fast_mixing_instructions+(ARM_OP_LEN*2) @ first NOP
```

Comment:
are you that this is correct?

Reply:
The original label pointed after the `LDM` and `MUL`, which is now part of the macro, so I have to skip the first two